x/operator/keeper/grpc_query.go

```diff
@@ -172,3 +172,36 @@ func (k Keeper) QueryAllOperatorConsAddrsByChainID(
 		Pagination:        pageRes,
 	}, nil
 }
+
+func (k *Keeper) QueryOperatorUSDValue(ctx context.Context, req *operatortypes.QueryOperatorUSDValueRequest) (*operatortypes.DecValueField, error) {
+	c := sdk.UnwrapSDKContext(ctx)
+	usdValue, err := k.GetOperatorUSDValue(c, req.Details.OperatorAddr, req.Details.AVSAddress)
+	if err != nil {
+		return nil, err
+	}
+	return &operatortypes.DecValueField{
+		Amount: usdValue,
+	}, nil
+}
+
+func (k *Keeper) QueryAVSUSDValue(ctx context.Context, req *operatortypes.QueryAVSUSDValueRequest) (*operatortypes.DecValueField, error) {
+	c := sdk.UnwrapSDKContext(ctx)
+	usdValue, err := k.GetAVSUSDValue(c, req.AVSAddress)
+	if err != nil {
+		return nil, err
+	}
+	return &operatortypes.DecValueField{
+		Amount: usdValue,
+	}, nil
+}
+
+func (k *Keeper) QueryOperatorSlashInfo(ctx context.Context, req *operatortypes.QueryOperatorSlashInfoRequest) (*operatortypes.QueryOperatorSlashInfoResponse, error) {
+	c := sdk.UnwrapSDKContext(ctx)
+	slashInfo, err := k.AllOperatorSlashInfo(c, req.Details.OperatorAddr, req.Details.AVSAddress)
```

Comment:
```suggestion
	slashInfo, err := k.AllOperatorSlashInfo(c, req.Details.AVSAddress, req.Details.OperatorAddr)
```

Reply:
Thanks, this RPC will be changed to consider the pagination.